compiler/rustc_metadata/src/rmeta/mod.rs

```diff
@@ -397,12 +397,14 @@ define_tables! {
     // That's why the encoded list needs to contain `ModChild` structures describing all the names
     // individually instead of `DefId`s.
     module_children_reexports: Table<DefIndex, LazyArray<ModChild>>,
+    ambiguity_module_children_reexports: Table<DefIndex, LazyArray<AmbiguityModChild>>,
```

Comment:
```suggestion
    ambiguity_module_children: Table<DefIndex, LazyArray<ModChild>>,
```
The distinction between reexports and non-reexports is not important here.
Publicly reexported ambiguities are very rare, so we don't need to care about performance and can encode everything  as `ModChild`ren.

Reply:
>Ambiguities that are not imports actually exist

*But* we should be able to get rid of them.
The ambiguity pair involves at least one (glob) import, and we can set it as the primary binding, then all ambiguity bindings will be imports.
(This is not a task for this PR, I'll try to do it separately.)